src/test/java/com/stripe/functional/ChargeTest.java

```diff
@@ -152,11 +153,8 @@ public void testMarkSafeWithTypedParams() throws StripeException {
   }
 
   @Test
-  public void testSearch() throws Exception {
-    ChargeSearchResult result =
-        Charge.search(ChargeSearchParams.builder().setQuery("currency:\"USD\"").build());
-    assertEquals(1, result.getData().size());
-    assertEquals(1, result.getTotalCount());
+  public void testSearch() {
+    assertNull(System.getProperty("java.version"));
```

Comment:
This is to verify that we are actually running on the selected java version.

Reply:
Thanks for doing this to illustrate the change! Let's make sure to revert the test.

One thing that's confusing to me is the `Display version` step in GH also uses Gradle but always shows JDK 17.

Maybe we could change it to show the accurate runtime version here? Or print this as part of the test suite?

Reply:
Added test_home to the display version output:
<img width="556" alt="image" src="https://user-images.githubusercontent.com/99349468/160665269-1b22844a-8bfe-43ba-8a12-8f8dc9b0640e.png">